tasks/javascript.js

```diff
@@ -59,12 +59,11 @@ gulp.task('scripts:lint', () => gulp.src([`${pathBuilder.jsSrcDir}/**/*.js`, ...
     .pipe(eslint({ fix: true }))
     .pipe(gulp.dest(file => file.base))
     .pipe(eslint.format())
-    .pipe(eslint.failAfterError())
-);
+    .pipe(eslint.failAfterError()));
 
 
 const jestTestRun = (args = {}) => jest.runCLI(
-    Object.assign({ bail: config.isProduction, passWithNoTests: true }, args),
+    { bail: config.isProduction, passWithNoTests: true, ...args },
```

Comment:
Be careful with this one, there were issues when using object spread recently, see #126 for details.